extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -116,9 +156,34 @@ function ($, mw, loader, nirvana, labeling) {
 	}
 
 	function processSave(modalInstance) {
-		var templateType = $('#TemplateClassificationEditForm [name="template-classification-types"]:checked').val();
+		var newTemplateType = $('#TemplateClassificationEditForm [name="template-classification-types"]:checked').val(),
+			oldTemplateType = '';
+
+		if ( $preselectedType.length > 0 ) {
+			oldTemplateType = $preselectedType.val();
+		}
+
+		if ( newTemplateType !== oldTemplateType ) {
+			// Track - modal saved with changes
+			tracker.track({
+				trackingMethod: 'both',
+				category: 'template-classification-dialog',
+				action: 'close',
+				label: 'changed',
+				value: newTemplateType
+			});
 
-		saveHandler(templateType);
+			saveHandler(newTemplateType);
+		} else {
+			// Track - modal saved without changes
+			tracker.track({
+				trackingMethod: 'both',
```

Comment:
in general what about creating tracking function within extension that will keep values that are common for all tracking calls here?
 ```
track = tracker.buildTrackingFunction({
	action: tracker.ACTIONS.CLICK,
	category: 'template-classification-dialog',
	trackingMethod: 'analytics'
})
```
and then 
``track({label: 'save-error'});``